packages/app-project/src/screens/ClassifyPage/ClassifyPageContainer.js

```diff
@@ -16,7 +17,12 @@ function storeMapper (stores) {
 class ClassifyPageContainer extends Component {
   render () {
     return (
-      <ClassifyPage {...this.props} />
+      <React.Fragment>
+        <ClassifyPage {...this.props} />
+        <CollectionsModal
+          subjectId='123'
```

Comment:
This should be passed in from the calling component (Classifier or Recent Subjects.)

Reply:
I don't see organizationally a reason to put this here. Why not put it as a child of the `Box` in `ClassifyPage`?

Reply:
I think it was originally here because the container is stateful, but all the state is inside the modal now so it could move.